webanno/webanno
# #1603 - Modularized and configurable coloring

**What's in the PR**
- Better modularization of the coloring strategies
- Introduce coloring rules at the layer level
- Introduced layer traits
- More final fields in the visual model
- Currently only span layers are supported!

**How to test manually**
* Read the documentation on coloring rules
* Add a few coloring rules and see if they work as expected

**Automatic testing**

**Documentation**
